oracle/README.md

```diff
@@ -145,6 +145,42 @@ Follow the instructions below to configure this check for an Agent running on a
 
 2. [Restart the Agent][8].
 
+#### Skip default metrics
+ To skip default metric checks for an instance and only run a custom query with an existing metrics gathering user, insert the tag `skip_default_metrics` with a value. This allows a configured instance of the Oracle integration to skip the system, process, and tablespace metrics from running, and allows custom queries to be run without having the permissions described in the [Datadog user creation](#datadog-user-creation) section. If this configuration entry is omitted, the user you specify is required to have those table permissions to run a custom query.
+
+ ```yaml
+   init_config:
+
+   instances:
+
+       ## @param server - string - required
+       ## The IP address or hostname of the Oracle Database Server.
+       #
+     - server: localhost:1521
+
+       ## @param service_name - string - required
+       ## The Oracle Database service name. To view the services available on your server,
+       ## run the following query:
+       ## `SELECT value FROM v$parameter WHERE name='service_names'`
+       #
+       service_name: "<SERVICE_NAME>"
+
+       ## @param user - string - required
+       ## The username for the user account.
+       #
+       user: <USER>
+
+       ## @param password - string - required
+       ## The password for the user account.
+       #
+       password: "<PASSWORD>"
+
+       ## @param skip_default_metrics - string - optional
+       ## Set this parameter to any value if you want to skip default system,
+       ## process, and tablespace metrics checks. 
+       #
+       skip_default_metrics: true
```

Comment:
Can we please rename this to `only_custom_queries`?

Reply:
Yes i will make these changes.